spec/controllers/posts_spec.rb

```diff
@@ -1,35 +1,45 @@
 require 'rails_helper'
 
-RSpec.describe 'Posts', type: :request do
-  describe 'GET "/users/1/posts"' do
-    before(:example) { get '/users/1/posts' }
+RSpec.describe "Posts", type: :request do
+  before(:each) do
+    @user = User.create(name: 'Tadesse', photo: 'https://unsplash.com/photos/mateopic',
+                        bio: 'Comedian, Ethiopian, stylish, I like football', posts_counter: 5)
+    @post = Post.create(author: @user, title: 'My first post', text: 'This is my first post', comments_counter: 1,
+                        likes_counter: 1)
+    @post2 = Post.create(author: @user, title: 'My second post', text: 'This is my second post', comments_counter: 1,
+                         likes_counter: 1)
+  end
+
+  describe "request list of all posts" do
+    before(:each) { get user_posts_path(@user) }
```

Comment:
- [ ] Good job adding some tests but please kindly extract the Post index tests from the show tests into separate files. Also, the following tests are missing

### Post Index Page
* Tests that asserts you can see the user's profile picture.
* Tests that asserts you can see the user's username.
* Tests that asserts you can see the number of posts the user has written.
* Tests that asserts you can see a post's title.
* Tests that asserts you can see some of the post's body.
* Tests that asserts you can see the first comments on a post.
* Tests that asserts you can see how many comments a post has.
* Tests that asserts you can see how many likes a post has.
* Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
* Tests that redirects to the post's show page when you click on a post.

### Post Show Page
* Tests that asserts you can see the post's title.
* Tests that asserts you can see who wrote the post.
* Tests that asserts you can see how many comments it has.
* Tests that asserts you can see how many likes it has.
* Tests that asserts you can see the post body.
* Tests that asserts you can see the username of each commentor.
* Tests that asserts you can see the comment each commentor left.

Reply:
Hello reviewer, thank you for the feedback, sorry to say this but you didn't check that we have created the test. 
![image](https://user-images.githubusercontent.com/85793292/206030318-48f0e031-e6e5-4d10-a5b4-6b0c44c387f9.png)
please check it again.